tensorflow/compiler/mlir/tensorflow/transforms/lower_tf.td

```diff
@@ -457,3 +457,49 @@ def LowerIsFiniteOp : Pat<(TF_IsFiniteOp $x),
                           (TF_SubOp $x, $x),
                           (TF_ConstOp (GetScalarOfType<0> $x)),
                           /*incompatible_shape_error*/ConstBoolAttrTrue)>;
+
+//===----------------------------------------------------------------------===//
+// Selu op patterns.
+//===----------------------------------------------------------------------===//
+
+// Selu(x) = scaledAlpha * (exp(x) - 1) if x < 0 else scale * x. Here
+// scale is taken as a constant 1.0507009873554804934193349852946 and
+// scaledAlpha  is taken as a constant 1.7580993408473768599402175208123.
+
+def getScale : NativeCodeCall<
+  "GetScalarOfType(getElementTypeOrSelf($0), 1.0507009873554804934193349852946)">;
+
+def getScaledAlpha : NativeCodeCall<
+  "GetScalarOfType(getElementTypeOrSelf($0), 1.7580993408473768599402175208123)">;
+
+
+def LowerSeluOp : Pat<(TF_SeluOp: $src_op TF_FloatTensor:$features),
+                  (TF_SelectOp
+                    (TF_GreaterEqualOp
```

Comment:
this should be greater instead of GreaterEqual

https://github.com/tensorflow/tensorflow/blob/818120e208d2b5622e7988f902a58ab5df3b69d0/tensorflow/compiler/tf2xla/kernels/elu_op.cc#L70